portfolio/src/main/webapp/script.js

```diff
@@ -92,3 +92,9 @@ function drawFoodChart() {
         foodChart.draw(breakfastData, dimensions);
   });
 }
+
+function createMap() {
+  const map = new google.maps.Map(
+    document.getElementById('map'),
+    {center: {lat: 37.422, lng: -122.084}, zoom: 16});
```

Comment:
nit: A comment here saying what these coordinates are would be helpful for future readers.

Reply:
Done